src/components/splitButton/splitButton.stories.js

```diff
@@ -0,0 +1,117 @@
+import { html } from 'lit';
+import { action } from '@storybook/addon-actions';
+import '../icon';
+import chevronRightIcon from '@carbon/icons/es/checkmark--outline/20';
+import './index';
+import { createOptionsArray } from '../../common/helpers/storybook';
+import {
+  SPLIT_BTN_KINDS,
+  SPLIT_BTN_SIZES,
+  SPLIIT_BTN_ICON_POSITION,
+} from './defs';
+const createSelectOptions = (defs) => [null, ...createOptionsArray(defs)];
+
+export default {
+  title: 'Components/SplitButton',
```

Comment:
Can we add a space here so it's readable in the Storybook sidebar?

Reply:
sure.